src/Uno.UWPSyncGenerator/Generator.cs

```diff
@@ -31,6 +31,72 @@ abstract class Generator
 		private const string BaseXamlNamespace = "Windows.UI.Xaml";
 #endif
 
+		private static readonly string[] skipBaseTypes = new[] {
```

Comment:
This array was allocated every time `HasValidBaseType` is called. It's now allocated once.